ies/recipes/opsworks-undeploy.rb

```diff
@@ -0,0 +1,3 @@
+if is_aws
```

Comment:
```
unless is_aws
  return
end
```

@gilleyj maybe we do a quick test-run for un-deploy on a test stack? I can't find JSON output for it anywhere.